brainstormforce/ultimate-addons-for-gutenberg
# Fix: BuddyBoss Platform plugin conflicting with Content Timeline and Post Timeline.

### Description

- Trello - https://trello.com/c/NXzRFGhc/219-fix-conflict-between-content-timeline-and-buddyboss-platform

### Screenshots

- Issue - https://share.getcloudapp.com/YEuXxmp0

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

1. Enable BuddyBoss Platform
2. Add a new page or post
3. Add 'Content Timeline' block for Ultimate Addons
4. See the error mentioned above. Block is unusable.
5. checkout to `conflict-with-timeline-buddyboss` and refresh the page. Check issue is resolved or not.

### Checklist:
